server/go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/bitdecaygames/fireport/server/
+module github.com/bitdecaygames/fireport/server
```

Comment:
Having the slash here actually causes issues with how it resolves local project imports. Once I removed it, things worked as expected.